src/quick-lint-js/error.h

```diff
@@ -63,6 +63,17 @@
           NOTE(QLJS_TRANSLATABLE("const variable declared here"),              \
                declaration))                                                   \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_assignment_to_imported_variable, "E0185",                          \
```

Comment:
When I wrote the task, I had in mind that we would reuse E0003 error_assignment_to_const_variable, but customize its message based on `var_kind`.

Making a separate error code seems fine too. :+1:

Reply:
Thinking about it more, I think separate error codes makes more sense.